pages/[id]/FeatureFlags.js

```diff
@@ -0,0 +1,197 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import Page from '../../components/page';
+import Card, {
+  CardHeader,
+  CardButtonContainer,
+  CardTitle,
+  CardContent,
+} from '../../components/card';
+import Button from '../../components/button';
+import { COLOR, EDIT } from '../../constants';
+import {
+  getTimestamp,
+  subscribeToFlags,
+  formatDate,
+  updateFlags,
+  getHackathonPaths,
+  getHackathons,
+} from '../../utility/firebase';
+import { useAuth } from '../../utility/auth';
+
+const CancelText = styled.p`
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin: 0px;
+`;
+
+const CancelButton = styled.button`
+  font-size: 16px;
+  cursor: pointer;
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin-left: 675px;
+  margin-right: 40px;
+  border: none;
+  outline: none;
+  background-color: ${COLOR.BACKGROUND};
+`;
+
+const FeatureFlagsContainer = styled.div`
+  display: flex;
+  width: 40%;
+  border: 2px solid lightgray;
```

Comment:
A bit of a nit but this would make the box look more consistent with the rest of the UI for FAQ and spocos
```suggestion
  border: 1px solid lightgray;
  border-radius: 3px;
```